README.md

```diff
@@ -214,8 +214,30 @@ numbers.Application.get(id, (err, app) => {
 
 ## Available Numbers
 
+### Available Numbers Area Code
+
+```Javascript
+var res = await numbers.AvailableNumbers.listAsync({areaCode:"919", quantity:"5"});
+console.log(res);
+```
```

Comment:
Huh. The rest of the sdk readme defaults to callback syntax instead of promise, so this seems somewhat out of the blue. Maybe transitioning to promise-based makes more sense but if APIMatic is coming up it might not be worth that effort?

Reply:
The task wanted the async order using await. I have nothing against putting in the callback sample if it'll make it more clear

Reply:
Nah, it was more of a small nitpick for consistency. I think it's pretty straightforward and probably wouldn't be the end of the world if you didn't include the callback